portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +25,4 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+const interval = setInterval(addRandomGreeting, 2500);
```

Comment:
Was the interval required? Maybe leave a comment as to why you have the setinterval.

Reply:
I wanted to change the random greeting every 2.5 seconds, so I added the interval. I've now added a comment explaining this behaviour.